README.md

```diff
@@ -54,33 +65,94 @@ This section describes the contributed pseudo debugger types and their support t
+
+1. The application's CSolution was initially built with a CMSIS-Toolbox version prior to v2.8.0 which is the first version to generate `*.cbuild-run.yml` files.
+1. You are using an [Arm CMSIS Solution](https://marketplace.visualstudio.com/items?itemName=Arm.cmsis-csolution) prior to v1.52.0 which is the first version to fully support the `${command:cmsis-csolution.getCbuildRunFile}` command.
+
+**Workarounds/Solutions**:
+
+1. Update the CMSIS Toolbox to the latest version. Additionally, you may have to run `cbuild setup --update-rte` in a terminal for a first-time generation of `*.cbuild-run.yml` file in an existing workspace.
+1. Update to Arm CMSIS Solution v1.52.0. Alternatively, replace `${command:cmsis-csolution.getCbuildRunFile}` with the path to the `*.cbuild-run.yml` in your workspace (`cmsis`>`cbuildRunFile` debug configuration setting).
+
+### AXF files built with Arm Compiler 6 toolchain
+
+When I download an AXF file built with Arm Compiler 6 I see the following warning and my application does not execute correctly. This happens regardless of the selected GDB server.
+```
+warning: Loadable section "RW_RAM0" outside of ELF segments
+  in /path/to/my/application.axf
+```
+
+**Possible Reason**: `arm-none-eabi-gdb` does not correctly load ELF program segments due to the way that Arm Compiler 6 generates section and program header information when scatterloading is used.
+
+**Workaround**: You can generate a HEX file for the program download, and the ELF file for debug purposes only. The following steps are required if you build a [CSolution](https://open-cmsis-pack.github.io/cmsis-toolbox/build-overview/)-based application with the [CMSIS-Toolbox](https://open-cmsis-pack.github.io/cmsis-toolbox/):
+
+1. Edit the `*.cproject.yml` file(s) of your application.
+1. Modify the [`output:type:`](https://open-cmsis-pack.github.io/cmsis-toolbox/YML-Input-Format/#output) node to generate both an `elf` and a `hex` file:
+```
+  output:
+    type:
+      - elf
+      - hex  
+```
+1. Build the solution.
+1. Keep the default configuration's `program` setting as is.
+```
+"program": "${command:cmsis-csolution.getBinaryFile}",
+```
+1. Modify the default debug configuration's `initCommands` list, so that the `load` command gets the relative path to the generated HEX file.
+```
+            "initCommands": [
+                "load ./relative/path/to/my/application.hex",
+                "break main"
+            ],
+```
 
-**Note**: The generated default debug configuration uses `JLinkGDBServer` as `target`.`server` setting. The executable with this name has slightly differing behavior depending on your host platform. It launches a GUI-less server on Linux and macOS. Whereas a GDB server with GUI is launched on Windows®. Please change the value to `JLinkGDBServerCL` to suppress the GUI on Windows.
+The debugger then loads the debug information from the ELF file. But uses the HEX file instead of the ELF file for program download.
```

Comment:
Rephrase: "This instructs the debugger to load the debug information from the ELF file and to use the HEX file for program download."